desktop/dev-environments.md

```diff
@@ -56,6 +56,22 @@ You can launch the application by running the command `make run` in your VS Code
 
 ![Single container Dev environment](/images/dev-env-localhost.png){:width="700px"}
 
+### Create a Dev Environment from a specific branch or tag
+
+You can create a dev environment from a specific branch (for example, a branch corresponding to a Pull Request) or a tag by adding `@mybranch` or `@tag` as a suffix to your Git URL:
+
+ `https://github.com/dockersamples/single-dev-env@mybranch`
+
+ or
+
+ `git@github.com:dockersamples/single-dev-env.git@mybranch`
+
+Docker then clones the repository with your specified branch or tag.
+
+> **Note**
+>
+> Known bug: when using `https://` Git URLs, you need to remove the `.git` suffix at the end of the URL if you want to specify a branch with `@mybranch`.
```

Comment:
```suggestion
> When cloning a Git branch using `https://`, you must remove the `.git` suffix at the end of the URL. For example, to clone `mybranch` on the `single-dev-env` Git repo, you must use `https://github.com/dockersamples/single-dev-env@mybranch`.
```

Reply:
OK. not calling this a known bug, it feels a bit strange that you need to remove the .git suffix for https, but not for ssh urls. Should we at least mention this is just "for now", and we’ll improve this later on ? 

Also, I didn’t mention an example, as this is the exact same one mentioned just above, not sure it’s worth repeating ?

Reply:
Could I rephrase: 
`Known bug: when cloning a Git branch using `https://`, you must remove the `.git` suffix at the end of the URL` ?

Reply:
(or "known limitation" ? )

Reply:
> "Could I rephrase:
Known bug: when cloning a Git branch using https://, you must remove the .git suffix at the end of the URL...

Sounds good. Can we change Known bug to "Known issue"?

Reply:
updated with "known issue" as suggested